web/frontend/src/components/slam.vue

```diff
@@ -95,6 +98,66 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMove = async () => {
+  moveClick = false;
+
+  const req = new motionApi.MoveOnMapRequest();
+
+  /*
+   * set request name
+   * here we set the name of the motion service the user is using
+   */
+  req.setName('builtin');
```

Comment:
[minor] for clarity please make this motionReq or motionServiceReq (comment can be removed then as well)

Reply:
you got it ;)